operations/research-and-development/product/development-process/new-bug-tickets/README.md

```diff
@@ -18,8 +18,10 @@ Bugs are any “obvious errors” on how the product or a feature is functioning
 
 ## 4. Information needed on Mattermost bug tickets
 
-* **Labels:** Add a `customer-bug` label if the ticket is based on a customer bug report. Add a `community-bug` label if the ticket is based on a community bug report.
-* **Environment:** There is an ``Environment`` field with drop-down options for whether the bug was found in Cloud test servers, Cloud production servers, Self-Hosted test servers, Self-Hosted production servers, or in Master/PR testing. Additionally, in the ticket description you can add details on Mattermost server and version, OS and version, Mattermost mobile app version, Mattermost desktop app version, any notable Mattermost configurations \(such as HA, Elasticsearch, image proxy, SSO\).
+* **Labels:** Add a label based on the following criteria:
+  - `prerelease-bug` label if the ticket is based on a bug found on master/main or during day-to-day use on community.mattermost.com.
+  - `rc-bug` label if the ticket is based on a bug found during release testing.
+  - `customer-bug` or `community-bug` label if the ticket is based on a customer or a community bug report on production.
```

Comment:
In addition, I wonder if we could include the points as listed by Linda at https://community.mattermost.com/private-core/pl/7q9c3qd9cpdk3rwix5a8yiwjjw